genbadge/utils_flake8.py

```diff
@@ -104,7 +105,8 @@ def get_flake8_stats(flake8_stats_file):
     return parse_flake8_stats(flake8_stats_txt)
 
 
-RE_TO_MATCH = re.compile(r"([0-9]+)\s+([A-Z0-9]+)\s.*")
+RE_TO_MATCH = re.compile(r"([0-9]+)\s+([A-Z0-9]+):*\s.*")
```

Comment:
```suggestion
# Note: the optional colon `:*` is to comply with some flake8 plugins, see PR #21
RE_TO_MATCH = re.compile(r"([0-9]+)\s+([A-Z0-9]+):*\s.*")
```

Reply:
Thank you for reviewing this @smarie.  I don't see why the very small patch related to the colon shouldn't be merged.

To be certain that this different format was the exception rather than some new format, I just did a quick review of the three plugins I found that have the colon:

[flake8-comments 0.1.2](https://github.com/orsinium-labs/flake8-comments) Was last updated 5 months ago, no comments or PRs.
[flake8-type-annotations 0.4.0](https://github.com/sobolevn/flake8-type-annotations) Is active with a few comments and PRs.

To my surprise:

[flake8-type-annotations 0.1.0](https://github.com/sobolevn/flake8-type-annotations) Is actually DEPRECATED, because the functionality was merged into flake8 under different rules, so I will stop using this one.

Reply:
> I don't see why the very small patch related to the colon shouldn't be merged.

It should ! The suggestion I made is about adding a comment above the line. For some reason github does not highlight the diff correctly.
Thanks for investigating this list of colon-creating plugins